translations/translations.json

```
@@ -1134,6 +1134,7 @@
       "networkRequestFailed": "The request could not be processed.",
       "responseMappingFailed": "An error occurred while processing the server's response.",
       "unknownError": "An unknown error occurred.",
+      "linkError": "Link could not be opened",
```

Please add a german translation if you want to use a custom translation. However, I would just use `unknownError` instead.
```suggestion
      "linkError": "Link could not be opened.",
```

---

i take the unknow error and remove this line